rollout/replicaset.go

```diff
@@ -120,10 +119,53 @@ func (c *rolloutContext) reconcileNewReplicaSet() (bool, error) {
 	if err != nil {
 		return false, err
 	}
+
+	abortScaleDownDelaySeconds := time.Duration(defaults.GetAbortScaleDownDelaySecondsOrDefault(c.rollout))
+	if c.isScaleDownOnabort() {
+		c.log.Info("scale down on abort for the newRS")
+
+		// if the newRS has scale down annotation and to be scaled down now, set
+		// Update newReplicasCount = int32(0)
+		if scaleDownAtStr, ok := c.newRS.Annotations[v1alpha1.DefaultReplicaSetScaleDownDeadlineAnnotationKey]; ok {
+			c.log.Infof("New rs '%s' has scaledown delay annotation: %s", c.newRS.Name, scaleDownAtStr)
```

Comment:
The comment says "rs has **delay** annotation, but the previous line checks for **deadline** annotation (DefaultReplicaSetScaleDownDeadlineAnnotationKey). Is that correct?

Reply:
The `DefaultReplicaSetScaleDownDeadlineAnnotationKey` is correct. Updated the log message to make it clear.